code/Extension/UserFormFieldEditorExtension.php

```diff
@@ -77,7 +78,11 @@ public function getFieldEditorGrid()
         $editableColumns->setDisplayFields([
             'ClassName' => function ($record, $column, $grid) use ($fieldClasses) {
                 if ($record instanceof EditableFormField) {
-                    return $record->getInlineClassnameField($column, $fieldClasses);
+                    $field = $record->getInlineClassnameField($column, $fieldClasses);
+                    if ($record instanceof EditableFileField) {
+                        $field->setAttribute('data-folderconfirmed', $record->FolderConfirmed ? 1 : 0);
```

Comment:
Would it not make more sense to have this bit of logic directly on `EditableFileField`?

Reply:
Nope, because that logic isn't used there.